src/Kris/LaravelFormBuilder/FormBuilder.php

```diff
@@ -48,6 +50,15 @@ public function create($formClass, array $options = [], array $data = [])
             ->setFormOptions($options)
             ->addData($data);
 
+        // reset model to match named form logic
+        if ($form->getName()) {
+            if ($form->getModel() instanceof Model && method_exists($form->getModel(), 'toArray')) {
+                $form->setModel([$form->getName() => $form->getModel()->toArray()]);
+            } elseif (is_array($form->getModel())) {
```

Comment:
We also need to check if model maybe already have that namespace, so we do not double it.